xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,487 @@
+               <emphasis role="strong">not</emphasis> need to install the
+               &hasi; pattern.
+            </para>
+         </step>
+         <step>
+            <para>
+               Log in to the &qnet; server and install the following package:
+            </para>
+            <screen>&prompt.root;<command>zypper</command> install corosync-qnetd</screen>
+         </step>
+         <step>
+            <para>
+               Start and enable the service:
+            </para>
+            <screen>&prompt.root;<command>systemctl</command> start corosync-qnetd
+&prompt.root;<command>systemctl</command> enable corosync-qnetd</screen>
+         </step>
+      </procedure>
+      <para>
+         Your &qnet; server is ready to accept connections from a &qdevice; client.
+         Further configurations are not needed.
+      </para>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-qdevice">
+      <title>Connecting &qdevice; Clients to the &qnet; Server</title>
+      <para>
+         After you have set up your &qnet; server, you are ready to connect the clients.
+         You can connect the clients during the installation of your cluster
+         or you can add them later. In the following procedure we use
+         the latter approach. We assume two cluster nodes (&node1; and &node2;)
+         which are already part of a two-node cluster and the &qnet; server (&node3;).
+      </para>
+      <procedure>
+         <remark>toms 2020-05-11: Is step 1 and step 2 really needed? Can I just
+         jump to step 3?</remark>
+         <step>
+            <para>
+               On &node1;, initialize your cluster:
+            </para>
+            <screen>&prompt.root;<command>crm</command> cluster init -y</screen>
+         </step>
+         <step>
+            <para>
+               On &node2;, join the cluster:
+            </para>
+            <screen>&prompt.root;<command>crm</command> cluster join -c &node1; -y</screen>
+         </step>
+         <step>
+            <para>
+               On &node1; and &node2;, bootstrap the <literal>qdevice</literal> stage.
+               Usually, in most cases the default settings are fine.
+               Provide at least <option>--qnetd-hostname</option> and the hostname or
+               IP address of the &qnet; server (&node3; in our case):
+            </para>
+            <screen>&prompt.root;<command>crm</command> cluster init qdevice --qnetd-hostname=&node3;</screen>
+            <para>
+               If you want to change the default settings, get a list of all
+               possible options with the command <command
+                  >crm cluster init qdevice --help</command>.
+               All options related to &qdevice; start with
+               <option>--qdevice-<replaceable>NAME</replaceable></option>.
+            </para>
+         </step>
+      </procedure>
+      <para>
+         If you have used the default settings, it will create a &qdevice; with TLS enabled
```

Comment:
_it would create_

Reply:
I think both is wrong, we need to use present tense here: "... it creates a ...". :wink: